### DISCUSSION TITLE
Issue Support web data not sync

### DISCUSSION THREAD

**Discussion by amanfathulla** [community/community#144889] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi..can u help me..i create system 90 % ready complete, data already save firebase, but have issue ,

if I log in, the saved data is not the same, the data is not synced, tell me what I need to do, if I log in on a different device, the saved data is not the same

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @amanfathulla, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it to General for you. Good luck!
